packages/reference/src/index.tsx

```diff
@@ -21,3 +21,5 @@ export type {
   MissingEntityCardProps,
   RenderCustomMissingEntityCard,
 } from './common/customCardTypes';
+export { isValidImage } from './utils/isValidImage';
+export { fromFieldValidations } from './utils/fromFieldValidations';
```

Comment:
A similar function is also exported from `field-editor-shared` did you try it?

Reply:
Let me check! :)

Reply:
Hum, actually maybe we should move `isValidImage` to `field-editor-shared`?

Reply:
It's only used in `isValidImage` but also doesn't seem tied to the reference logic. I'm fine with that.

The two implementations of `fromFieldValidations` seem to be slightly different. Not sure which one is more correct/complete.

Reply:
I think I don't need `fromFieldValidations`, right now I'm only using `isValidImage`.

Reply:
Let's not export it then